checks/webhook.go

```diff
@@ -46,7 +46,7 @@ func WebHooks(c *checker.CheckRequest) checker.CheckResult {
 		})
 
 		e := sce.WithMessage(sce.ErrorUnsupportedCheck, "SCORECARD_V6 is not set, not running the Webhook check")
-		return checker.CreateRuntimeErrorResult(CheckWebHooks, e)
```

Comment:
I think it makes sense for this to be `RuntimeError`. What should change is the behavior that this check runs by default (I assume that's why you updated this?).

Reply:
Correct. It ran by default, so every run was exiting with a non-zero status. I'll update the PR to revert this change, and instead disable the check by default.